src/components/RemoteImage.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
```

Comment:
I think we might be able do do away with all of this component (this is a bit of a legacy one from the client) and just use `useImageExists()`

Reply:
Done https://github.com/aragon/court-dashboard/pull/76/commits/1adf2a71f8227155306915a4fbff008083a8f4ed